### DISCUSSION TITLE
Welcome to volesti Discussions!

### DISCUSSION THREAD

**Discussion by vissarion** [GeomScale/volesti#202] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shahpratham** [comment]
Hello everyone! Pratham Shah here. I'm a second-year undergraduate in Information Technology at VJTI, Mumbai. I went through the volesti codebase and would love to contribute to it. I have sound knowledge of C++ and maths algorithms needed in this. I would also like to see upcoming GSoC projects and participate in it. Hoping to do my best!

**Comment by rohitsalla** [comment]
Hello everyone , I'm Rohit Salla !! I am looking forward to contribute to this community. I have basic understanding on computational geomentry , statistics and c++.. 
recently i tried to steup environment in docker , it was showing error...can anyone help me out ??
